portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="ravenclaw_stripes">
+    <div id="content">
+        <h1 id="title"><strong>Alisa's Gallery</strong></h1>
```

Comment:
indent by 2 space at at time here and below, including the css file: https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
Added a commit which changes the indents to 2 spaces.